server/src/test/java/io/crate/lucene/BitStringQueryTest.java

```diff
@@ -64,17 +61,17 @@ public void test_eq_any_on_bits_uses_termset_query() throws Exception {
 
         try (var tester = builder.build()) {
             Query query = tester.toQuery("bits = ANY([B'01001110', B'11111111'])");
-            assertThat(query, instanceOf(TermInSetQuery.class));
-            assertThat(query.toString(), is("bits:(r [ff])"));
+            assertThat(query).isExactlyInstanceOf(TermInSetQuery.class);
+            assertThat(query.toString()).isEqualTo("bits:(r [ff])");
 
             List<Object> result = tester.runQuery("bits", "bits = ANY([B'01001110', B'11111111'])");
-            assertThat(result, Matchers.empty());
+            assertThat(result).isEmpty();
 
             result = tester.runQuery("bits", "bits = ANY([B'00010110', B'11111111'])");
-            assertThat(result, Matchers.contains(bits1));
+            assertThat(result).contains(bits1);
```

Comment:
Here and all the places you replace `Matchers.contains()`, it needs to be `containsExactly`.